nox/sessions.py

```diff
@@ -432,7 +432,9 @@ def install(self, *args: str, **kwargs: Any) -> None:
 
         self._run("python", "-m", "pip", "install", *args, external="error", **kwargs)
 
-    def notify(self, target: "Union[str, SessionRunner]") -> None:
+    def notify(
+        self, target: "Union[str, SessionRunner]", posargs: Optional[List[str]] = None
```

Comment:
Maybe choose a generic argument type for the public API?

```suggestion
        self,
        target: "Union[str, SessionRunner]",
        posargs: Optional[Iterable[str]] = None,
```